ansible/README.test.md

```diff
@@ -83,6 +83,13 @@ ansible-playbook test_sonic.yml -i inventory --limit {DUT_NAME} --tags lag-2 --e
 ```
 - Requires switch connected to a VM testbed with lag configured (t0, t1-lag)
 
+### Lag-fallback test
+```
+ansible-playbook test_sonic.yml -i inventory --limit {DUT_NAME} --tags lag-2 --extra-vars "testbed_type={TESTBED_TYPE} ptf_host={PTF_HOST}"
```

Comment:
You need to convert your test into new format
`ansible-playbook test_sonic.yml -i str --limit str-s6100-acs-1 --vault-password-file password.txt -e testcase_name=lag-2 -e testbed_name=vms3-t0-s6100`

Please check how the recently added tests are executed. Please let me know if you have questions.

Reply:
sure. Thanks @pavel-shirshov 
will rewrite the test case